spec/features/inactive_users_spec.rb

```diff
@@ -1,8 +1,8 @@
 feature "Inactive users" do
   scenario "not included in approving official dropdown" do
-    inactive_approving_official = create(:user, :inactive)
-    active_approving_official = create(:user, :active)
-    user = create(:user)
+    inactive_approving_official = create(:user, :inactive, client_slug: 'ncr')
```

Comment:
there's a tremendous weight of history in `spec/` that uses single quotes. I'm happy to change these that I have introduced. Only 3k+ more to change.